### PR TITLE
Add support for multiple init scripts

### PR DESCRIPTION
**Pull request by sfodje**
This is in response to issue testcontainers/testcontainers-java#2232

### CODE REVIEW

File: modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java [hunk 1]
```diff
@@ -107,8 +107,8 @@ public SELF withConnectTimeoutSeconds(int connectTimeoutSeconds) {
         return self();
     }
 
-    public SELF withInitScript(String initScriptPath) {
-        this.initScriptPath = initScriptPath;
+    public SELF withInitScript(String... initScriptPath) {
```
Comment (bsideup) [inline modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java:110]: this breaks binary compatibility. Please make sure that the old method (`withInitScript(String)`) works as before

Comment (sfodje) [reply modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java]: Hello @bsideup,
Thanks for your feedback.
I've merged some changes from @jbek7 into this pull request that introduces a separate method `withMultiInitScript` to accept multiple sql files.

Comment (png-tech) [reply modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java]: @sfodje @bsideup
thanks.

File: modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java [hunk 2]
```diff
@@ -108,7 +108,11 @@ public SELF withConnectTimeoutSeconds(int connectTimeoutSeconds) {
     }
 
     public SELF withInitScript(String initScriptPath) {
-        this.initScriptPath = initScriptPath;
+        return withMultiInitScript(initScriptPath);
+    }
+
+    public SELF withMultiInitScript(String... initScriptPaths) {
```
Comment (rnorth) [inline modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java:114]: Just a thought but... If the parameters were `String, String...` couldn't we keep the original method name and avoid problems with overloading/binary incompatibility? 🤔

File: modules/database-commons/src/main/java/org/testcontainers/ext/ScriptUtils.java [hunk 3]
```diff
@@ -285,27 +286,47 @@ public static boolean containsSqlScriptDelimiters(String script, String delim) {
...
 	/**
-	 * Load script from classpath and apply it to the given database
+	 * Loads multiple scripts from classpath and applies it to the given database
 	 *
 	 * @param databaseDelegate database delegate for script execution
-	 * @param initScriptPath   the resource to load the init script from
+	 * @param initScriptPaths   the resources to load the init scripts from
 	 */
-	public static void runInitScript(DatabaseDelegate databaseDelegate, String initScriptPath) {
+	public static void runMultiInitScript(DatabaseDelegate databaseDelegate, String... initScriptPaths) {
```
Comment (rnorth) [inline modules/database-commons/src/main/java/org/testcontainers/ext/ScriptUtils.java:305]: Do we need such big changes to `ScriptUtils` for this? It looks like [this other PR](https://github.com/testcontainers/testcontainers-java/pull/1997/files#diff-a3c7df4583fc2f224a7951d9235c8ab1R247) has a simpler approach, and I can't see why that wouldn't work...

File: modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java (hunk 2 above)
Comment (sfodje) [reply modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java:114]: I had the same thought, but assumed the purpose was to keep the API visually and functionally the same. Now that you raise it up, it is a good question. The previous iteration was a lot simpler and less complicated.

**Comment by jitendramandal92** [comment]
Hello @rnorth/@bsideup/@kiview,
I tried to overcome this issue by overriding ```runInitScriptIfRequired``` of JdbcDatabaseContainer class for supporting multiple script files, Invoking ScriptUtils.runInitScript() for each script file. It worked locally but failed in CI Jenkins with error not able to create JDBC connection. However, for single file it worked in CI also. Any update on this ticket?

**Pull request closed without merging**